python/fledge/plugins/south/person_detection/person_detection.py

```diff
@@ -59,28 +59,28 @@ def check_background():
     'model_file': {
         'description': 'TFlite model file to use for inference',
         'type': 'string',
-        'default': 'detect.tflite',
+        'default': 'detect_edgetpu.tflite',
         'order': '1',
-        'displayName': 'TFlite model file'
+        'displayName': 'TFlite Model File'
     },
     'labels_file': {
         'description': 'Labels file used during inference',
         'type': 'string',
         'default': 'coco_labels.txt',
         'order': '2',
-        'displayName': 'Labels file'
+        'displayName': 'Labels File'
     },
     'asset_name': {
-        'description': 'Asset Name',
+        'description': 'Asset name',
         'type': 'string',
-        'default': 'Detection Results',
+        'default': 'person_detection',
         'order': '3',
         'displayName': 'Asset Name'
     },
     'enable_edge_tpu': {
-        'description': 'Connect the Coral Edge TPU and enable this',
+        'description': 'Connect the Coral Edge TPU and ensure this is enabled in order to use Edge TPU',
```

Comment:
May be change to "Connect Coral Edge TPU and select this option to use TPU for inference".